### PR TITLE
Website restructuring

### PR DESCRIPTION
**Pull request by lianmakesthings**
Draft for new website as discussed in this issue https://github.com/cncf/tag-app-delivery/issues/410
Discussion Doc: https://docs.google.com/document/d/1RibiFs3KlQ0FhfaF4X8owQmDzLuN2y4jRnQy3TYp6KI/edit

### CODE REVIEW

**Comment by joshgav** [comment]
Thank you @lianmakesthings! Reviewing now.

@cjyabraham can you please help us get Netlify previews working for this PR? The Netlify deploy logs state `Custom publish path detected. Proceeding with the specified path: 'website/public'` followed by `No changes detected in base directory. Returning early from build.` Perhaps what we need is to trigger Netlify rebuilds for anything in `website`? Thanks!

File: website/content/wgs/operator/charter/_index.md
Comment (joshgav) [inline website/content/wgs/operator/charter/_index.md:8]: We've been removing these lists of names because a) they're out of date - many of these parties are no longer interested and b) it misleads newcomers to thinking they need to submit PRs to add themselves.

My vote would be to remove the "Interested Parties" section here, particularly since this WG is a bit of a placeholder in any case.

File: website/content/wgs/platforms/_index.md
Comment (joshgav) [inline website/content/wgs/platforms/_index.md:15]: Please add:
`* Abby Bangser (@abangser)`

**Review by joshgav — COMMENTED** [review]
Thanks for this @lianmakesthings, it's a nice structure to build on.

LGTM modulo the comment about the list of names, and need to add Abby to the list of WG Platforms chairs.

Let's wait for @cjyabraham to fix the preview deployment before approving and merging.

**Comment by cjyabraham** [comment]
The netlify setup is already set to watch anything in /website/ and anything in /*-whitepaper/ when building so I'm not sure why this build was skipped. I just kicked it off again and it worked this time: https://deploy-preview-426--tag-app-delivery.netlify.app/

File: website/content/wgs/platforms/_index.md
Comment (joshgav) [reply website/content/wgs/platforms/_index.md:15]: Reminder to please add @abangser before merging, thanks!

**Review by joshgav — CHANGES_REQUESTED** [review]
The old URL now works! It would be nice if the new URL also worked, but we don't have to block on that for now.

So LGTM after the remaining comments are addressed - let's remove the list of names and please add @abangser to the list of chairs for Platforms.

Thanks again @lianmakesthings!

**Comment by lianmakesthings** [comment]
@joshgav updated the PR to address your comments.

Can you check if everything's in order? (weirdly, the deploy preview seemed to have failed again)

**Comment by cjyabraham** [comment]
I've re-triggered the deploy and it worked this time. I'm not sure why it keeps cancelling the deploy when you commit...

**Comment by lianmakesthings** [comment]
I will try to not take it personally 😂

**Comment by joshgav** [comment]
@thschue could you take a look too?

**Comment by cjyabraham** [comment]
FYI, I just did some testing to see why the preview instances were sometimes being skipped on this PR but I can't figure it out. I created [a PR from a branch on this repo](https://github.com/cncf/tag-app-delivery/pull/437) and the build worked. I then [created one from a fork](https://github.com/cncf/tag-app-delivery/pull/438) and it worked as well.

**Pull request merged**
